src/main/java/com/exactpro/th2/sim/impl/Simulator.java

```diff
@@ -116,6 +121,27 @@ public void init(InitializationContext context) {
             }
         });
 
+
+        this.messageBatcher = new MessageBatcher(
+                config.getMaxBatchSize(),
+                config.getMaxFlushTime(),
+                bookName,
+                MessageBatcher.GROUP_SELECTOR,
```

Comment:
```suggestion
                MessageBatcher.ALIAS_SELECTOR,
```
because you use session alias instead group.
Also, I think the current sim implementation lose infrormation about session group